### ISSUE TITLE
Why the accuracy of training increases dramatically at 100th epoch in the Figure 3(a) of the paper?

### ISSUE THREAD

**Issue by jeff62802217** [kenshohara/3D-ResNets-PyTorch#164]
I carried out the experiment again but found the training accuracy converged to 27% and remained unchanged.

**Comment by g2-bernotas** [comment]
I don't have an answer, but it could be that the NN found a more optimal solution at epoch 100. 
I have worked a bit with the Mask R-CNN before and I found that not all the time the best solution is found as the training, to my understanding, is a bit random. Or maybe the authors changed a parameter value at that epoch to result in such a dramatical change.
